server/server.js

```diff
@@ -48,8 +61,9 @@ module.exports = function(port, middleware, callback) {
         var id  = req.params.id;
         var todo = getTodo(id);
         if (todo) {
-            if (req.body.title) {
+            if (typeof req.body.title !== "undefined" && typeof req.body.isComplete !== "undefined") {
```

Comment:
I didn't put it in in the original commit (d29cc72) but found it was throwing trues left, right and center when testing. Might be because it's a not equal...